src/components/dataset/summary/row.vue

```diff
@@ -11,16 +11,20 @@ except according to the terms contained in the LICENSE file.
 -->
 <template>
   <div class="dataset-summary-row">
-    <!-- eslint-disable-next-line vuejs-accessibility/click-events-have-key-events, vuejs-accessibility/interactive-supports-focus -->
-    <div class="row" role="button" @click.prevent="toggleExpanded">
+    <div class="row">
       <div class="col-xs-6 dataset-name-wrap">
-        <div class="dataset-name text-overflow-ellipsis" v-tooltip.text>{{ dataset.name }}</div>
+        <div class="dataset-name text-overflow-ellipsis" v-tooltip.text>
+          <router-link :to="datasetPath(projectId, dataset.name)" v-tooltip.text>
```

Comment:
It'd be great if you could test this `to` prop in the tests of `DatasetSummaryRow`.